### PR TITLE
py38: Django 2.1

### PR DESCRIPTION
**Pull request by joshuarli**
Dependencies which will be separated into individual PRs: nothing this time.

Notable separated PRs:

- https://github.com/getsentry/sentry/pull/27874

Prerequisite testing:

- X-Django model pickling testing done, see https://github.com/getsentry/sentry/tree/joshuarli/test-x-django-pickles
- `LAST_VERIFIED_DJANGO_VERSION` verification done: https://github.com/getsentry/sentry/pull/27720

### CODE REVIEW

File: tests/sentry/web/frontend/test_auth_organization_login.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def test_flow_as_anonymous(self):
 
         user = auth_identity.user
         assert user.email == "foo@example.com"
-        assert not user.has_usable_password()
+        assert not user.password
```
Comment (joshuarli) [inline tests/sentry/web/frontend/test_auth_organization_login.py:88]: While all that's needed is just to fix this test I think we should not use `has_usable_password`. Mostly confident in this, need to look at `createuser` and the sudo powers assumed thing.

**Review by wedamija — APPROVED** [review]

**Comment by joshuarli** [comment]
Was able to reproduce the cloudbuild fail against local devserver with a slightly modified version of https://github.com/getsentry/onpremise/blob/master/_integration-test/run.sh

```
server   Forbidden (CSRF token missing or incorrect.): /auth/login/sentry/
 server   21:18:07 [WARNING] django.security.csrf: Forbidden (CSRF token missing or incorrect.): /auth/login/sentry/ (status_code=403 request=<WSGIRequest: POST '/auth/login/sentry/'> message='Forbidden (CSRF token missing or incorrect.): /auth/login/sentry/')
 server    403  POST /auth/login/sentry/ HTTP/1.1 6351
```

**Comment by joshuarli** [comment]
Django 2.0 is in red, 2.1 in green:

image

For an awk that will work with both, just need both `'` and `"` as field separators:

```
josh@thanks-again-tim-apple sentry * master $ cat 21 | awk -F '['\''"]' '
    /csrfmiddlewaretoken/ {
    print $4 "=" $6;
    exit;
  }'
csrfmiddlewaretoken=bfLznrUEOGMVilCn1wGl1WLzjCDcNDIqniDC9APwjOsdVziDGMvsb7N9dwp1wftv
josh@thanks-again-tim-apple sentry * master $ cat 20 | awk -F '['\''"]' '
    /csrfmiddlewaretoken/ {
    print $4 "=" $6;
    exit;
  }'
csrfmiddlewaretoken=17zEAMqhbj2ucf1ajH9OpXQZ8i0bh7Okm51Q8N2UptIbZy7yDsGRnsfZllS3YLzC
```

**Pull request merged**